bower.json

```diff
@@ -14,6 +14,7 @@
     "angular-route": "~1.3.0",
     "angular-sanitize": "~1.3.0",
     "angular-touch": "~1.3.0",
+    "angular-bootstrap": "~0.13.2",
```

Comment:
Do we have bootstrap CSS and JS on the page twice now? Can we remove the old one?